net/netatalk4/Portfile

```diff
@@ -0,0 +1,70 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+PortGroup           meson 1.0
+
+name                netatalk4
+conflicts           netatalk
+
+github.setup        Netatalk netatalk netatalk-4-1-2
+github.tarball_from archive
+revision            0
+
+checksums           rmd160  b6953e69144d6542c1b426015635d58f695cdba9 \
+                    sha256  012f2a32c4904f4010c1b924e5c01cf16dfc628cb32965be54479721f30aa30d \
+                    size    1327961
+
+categories          net
+license             GPL-2+
+maintainers         {openmaintainer github:trodemaster}
```

Comment:
this is not correct; withing the brackets it should be listing both the e-mail address and GitHub handle of the maintainer. `openmaintainer` is ideally added to allow others to make (minor) modifications to the Portfile without explicit approval.
In this case it should thus likely be: `maintainers         {@trodemaster example.org:julesverne} openmaintainer` where you add the correct obfuscated e-mail address.

Reply:
Ok I generated a new icloud private email and included it as specified.